app/views/posts/index.html.erb

```diff
@@ -1 +1,33 @@
-<h2>All the post of a user goes here</h2>
\ No newline at end of file
+<div class="posts-container">
+    <div class="post-header">
+        <div class="post-image-holder">
+            <img src="<%= @user.photo %>" alt="user">
+        </div>
+            <div class="post-details">
+                <h3><%= @user.name %></h3>
+                <p>Posts: <%= @user.posts_counter || 0%></p>
+        </div>
+    </div>
+
+    <% @posts.reverse.each_with_index do |post, index| %>
+        <div class="post">
+            <a href="/users/id/posts/<%= post.id %>"><h4>Post No: <%= index+1 %></h4></a>
+                
+            <h3><%= post.title %></h3>
+            <p><%= post.text %></p>
+            <div class="">
+                <p>Comments: <%= post.likes_counter || 0 %></p>
+                <p>Likes: <%= post.comments_counter || 0 %></p>
+
+                <div class="comments">
```

Comment:
- [ ] kindly add controllers add `likes` and `comments` controller :+1:

Reply:
@andersonlebon 
Hi, I think we don't need likes and comment controllers in this milestone, because they are not rendering individually on a page. They appear on a page only with Users or Posts. We will need Likes and comments controllers only to create, edit or delete a like or comment. However I can create comments and likes controllers not, but what action do I create there? Index and show? Reply at your earliest.